backend/src/main/java/wooteco/retrospective/config/AuthenticationPrincipalConfig.java

```diff
@@ -0,0 +1,39 @@
+package wooteco.retrospective.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import wooteco.retrospective.application.member.MemberService;
+import wooteco.retrospective.infrastructure.auth.JwtTokenProvider;
+import wooteco.retrospective.presentation.auth.AuthenticationPrincipalArgumentResolver;
+
+import java.util.List;
+
+@Configuration
+public class AuthenticationPrincipalConfig implements WebMvcConfigurer {
+
+    private final MemberService memberService;
+    private final JwtTokenProvider jwtTokenProvider;
+
+    public AuthenticationPrincipalConfig(MemberService memberService, JwtTokenProvider jwtTokenProvider) {
```

Comment:
```suggestion
    public AuthenticationPrincipalConfig(MemberService memberService, 
                                         JwtTokenProvider jwtTokenProvider) {
```
구글 컨벤션에 따라 100줄이 넘으면 개행 해주세요